utils/abilities.cfg

```diff
@@ -1891,7 +1924,7 @@ Enemy units cannot see this unit during dawn or dusk, except if they have units
         name= _ "electrical discharge"
         female_name= _ "female^electrical discharge"
         description= _ "When this unit is hit with a melee attack, the resulting discharge of electricity causes an equal amount of lightning damage to be taken by the attacker. It can kill."
-#        halo_image_self="halo/reflect.png"  # need something that looks like it's throwing off sparks
+        #        halo_image_self="halo/reflect.png"  # need something that looks like it's throwing off sparks
```

Comment:
Please open an issue for this once this change is merged.

Reply:
noted